run.py

```diff
@@ -132,4 +151,12 @@ def main():
         write_data_to_file(source_data, destination_dataset, upload_url)
 
 if __name__ == '__main__':
-    main()
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--dataset_id', dest='dataset_id', type=str, help='Source dataset id', required=True)
+    parser.add_argument('--table_name', dest='table_name', type=str, 
+        help='Source table name, also used to name destination dataset', required=True)
+    parser.add_argument('--installation_id', dest='target_installation_id', type=str, 
+        help='Target installation id (optional)', required=False)
```

Comment:
Given that we throw an error when there are multiple installations and this argument isn't passed, should we clarify in the help something along the lines of `Target installation id (optional when there is only one installation)`